estate/models/estate_property.py

```diff
@@ -0,0 +1,87 @@
+from odoo import api, models, fields
+from odoo.exceptions import UserError
+import datetime
+from dateutil.relativedelta import relativedelta
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property for the Real Estate app"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    notes = fields.Html()
+    date_availability = fields.Date(copy=False, default=datetime.date.today() + relativedelta(months=+3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(selection=[("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")])
+    active = fields.Boolean(default=True)
+    state = fields.Selection(selection=[("new", "New"), ("offer_received", "Offer Received"), ("offer_accepted", "Offer Accepted"), ("sold", "Sold"), ("cancelled", "Cancelled")], copy=False, required=True, default="new")
```

Comment:
```suggestion
    state = fields.Selection(selection=[
        ("new", "New"), 
        ("offer_received", "Offer Received"), 
        ("offer_accepted", "Offer Accepted"), 
        ("sold", "Sold"), 
        ("cancelled", "Cancelled"),
    ], copy=False, required=True, default="new")
```
Line is way too long

Reply:
Line is till too long here